10/jdk/alpine/Dockerfile.hotspot.releases.slim

```diff
@@ -19,24 +19,21 @@
 
 FROM alpine:3.7
 
-MAINTAINER Dinakar Guniguntala <dinakar.g@in.ibm.com> (@dinogun)
-
-RUN apk --update add --no-cache ca-certificates curl openssl binutils xz \
-    && GLIBC_VER="2.25-r0" \
-    && ALPINE_GLIBC_REPO="https://github.com/sgerrand/alpine-pkg-glibc/releases/download" \
-    && curl -Ls ${ALPINE_GLIBC_REPO}/${GLIBC_VER}/glibc-${GLIBC_VER}.apk > /tmp/${GLIBC_VER}.apk \
-    && apk add --allow-untrusted /tmp/${GLIBC_VER}.apk \
-    && curl -Ls https://www.archlinux.org/packages/core/x86_64/gcc-libs/download > /tmp/gcc-libs.tar.xz \
-    && mkdir /tmp/gcc \
-    && tar -xf /tmp/gcc-libs.tar.xz -C /tmp/gcc \
-    && mv /tmp/gcc/usr/lib/libgcc* /tmp/gcc/usr/lib/libstdc++* /usr/glibc-compat/lib \
-    && strip /usr/glibc-compat/lib/libgcc_s.so.* /usr/glibc-compat/lib/libstdc++.so* \
-    && curl -Ls https://www.archlinux.org/packages/core/x86_64/zlib/download > /tmp/libz.tar.xz \
-    && mkdir /tmp/libz \
-    && tar -xf /tmp/libz.tar.xz -C /tmp/libz \
-    && mv /tmp/libz/usr/lib/libz.so* /usr/glibc-compat/lib \
-    && apk del binutils \
-    && rm -rf /tmp/${GLIBC_VER}.apk /tmp/gcc /tmp/gcc-libs.tar.xz /tmp/libz /tmp/libz.tar.xz /var/cache/apk/*
+RUN apk --update add --no-cache ca-certificates curl openssl binutils xz
```

Comment:
why multiline `RUN` was changed to multiple `RUN` instructions ? This is something what should be avoid.

Reply:
no that's the whole point. It's better to have multiple `RUN` instructions because they can be individually executed and cached. If the build fails it can resume where it stopped.
Because the final image is created in a separate stage, the number of stages in the `prepare` stage will have no impact on the final image

Reply:
ok, can se an advantage in that, but even so I would divide it more sensible, to parts which can fail and which couldn't. For example 
`apk add --allow-untrusted /tmp/${GLIBC_VER}.apk`
`mkdir /tmp/libz`

For the other side `apk del binutils` has no sense in that form

Reply:
yes sorry, this was the first attempt and I didn't understand the dockerfile very well. I'll make another attempt soon which I hope will make more sense.

Reply:
nothing for sorry for ;) just trying to be helpful. 

But to be honest I wonder if the build will be faster enough to justify that (I am reffering to multi-`RUN` instructions)